### ISSUE TITLE
git2go is currently unmaintained

### ISSUE THREAD

**Issue by ReillyBrogan** [libgit2/libgit2#6849]
Posting here because there appears to be actual libgit2 developers/maintainers that read issues here. The git2go project appears to be completely unmaintained at the moment. There have been no commits in almost two years, it's [still stuck](https://github.com/libgit2/git2go/issues/951) on libgit2 1.5, and there's a [reported memory leak](https://github.com/libgit2/git2go/issues/959) with reproduction steps that should be addressed.

Considering that git2go is listed on the https://libgit2.org/ homepage and is ostensibly part of the libgit2 github organization this does not reflect well on the overall project. I think a call for maintainers should be put out or it should be removed from the website and a note added to the readme that it's unmaintained.

(side note, several of the other third-party bindings appear to be unmaintained as well. They should probably be pruned from the website).

**Comment by anatol** [comment]
Agreed, we actively using `git2go` and it is unfortunate to see the project is stagnating.

**Comment by ethomson** [comment]
Indeed. There have been a few conversations with folks who might be interested in stepping up, but ultimately, git2go needs a maintainer (or team of). I'm probably not the best person to do so (my go is... nearly nonexistent).

**Comment by AaronMegs** [comment]
> Agreed, we actively using `git2go` and it is unfortunate to see the project is stagnating.同意，我们积极使用 `git2go` ，但不幸的是看到该项目停滞不前。

We are the same. If there are like-minded people, please take action and continue to maintain it.
